pkg/runtime/wfengine/backends/actors/backend.go

```diff
@@ -343,8 +343,13 @@ func (abe *ActorBackend) GetActivityWorkItem(ctx context.Context) (*backend.Acti
 }
 
 // GetOrchestrationRuntimeState implements backend.Backend
-func (*ActorBackend) GetOrchestrationRuntimeState(context.Context, *backend.OrchestrationWorkItem) (*backend.OrchestrationRuntimeState, error) {
-	return nil, errors.New("not supported")
+func (abe *ActorBackend) GetOrchestrationRuntimeState(ctx context.Context, owi *backend.OrchestrationWorkItem) (*backend.OrchestrationRuntimeState, error) {
+	state, err := LoadWorkflowState(ctx, abe.actorRuntime, string(owi.InstanceID), abe.config)
```

Comment:
Any particular reason for implementing this?

Reply:
@cgillum this is required for cascade purge: we need to get list of sub-orchestrations to purge: https://github.com/microsoft/durabletask-go/blob/b58abfd02e48667ae6d8622de1531b322178b8b9/backend/backend.go#L135